### ISSUE TITLE
Flag statistics

### ISSUE THREAD

**Issue by matthiakl** [widelands/widelands#4266]
**Is your feature request related to a problem? Please describe.**
When I like to move large quantities of wares (e. g. from one warehouse to another), I always have to carefully observe the path for bottlenecks, where a flag got filled up.

**Describe the solution you'd like**
I would like to have a "Flags" tab in the building statistics window. Like for other buildings, it should show the amount of flags and those with low productivity (blocked), with the option to zoom to them.

**Describe alternatives you've considered**
As Flags are no real buildings, an alternative might be to send a message.

**Comment by gunchleoc** [comment]
How about only adding blocked flags?

Productive flags would create a lot of clutter.

**Comment by matthiakl** [comment]
> How about only adding blocked flags?

Yes, that would be sufficient, if it is possible to zoom to them.
Finding blocked flags has already been requested in #1781, however it doesn't allow to zoom directly to it.

**Comment by cake-pie** [comment]
I think this can be considered solved using different approach in #1781 + #5876
